README.md

```diff
@@ -1 +1,159 @@
+·················|···························|·············|·············|·············|··············|··············
+|  Contract      ·  Method                   ·  Min        ·  Max        ·  Avg        ·  # calls     ·  usd (avg)  │
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G2Helper · _bn128_g2_add             ·        -    ·        -    ·  56879      ·     8         ·  0.2       │ 
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G2Helper · _bn128_g2_multiply        ·  2947053    ·  3142700    ·  3060006    ·    15        ·  10.31      │
+·················|···························|·············|·············|·············|··············|··············
+|  Deployments                               ·                                         ·  % of limit  ·             │
+·············································|·············|·············|·············|··············|··············
+|  BN256G2Helper                             ·        -    ·        -    ·  214244     ·     3.2 %    ·   0.9       │
+·--------------------------------------------|-------------|-------------|-------------|--------------|-------------·
+```
+
+```
+·············································|···························|·············|·····························
+|  Methods                                   ·               20 gwei/gas               ·       213.78 usd/eth       │
+·················|···························|·············|·············|·············|··············|··············
+|  Contract      ·  Method                   ·  Min        ·  Max        ·  Avg        ·  # calls     ·  usd (avg)  │
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G1Helper ·  _add                     ·   26055     ·   27591     ·   26835     ·     12       ·   0.1       │
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G1Helper ·  _bn256CheckPairing       ·  143777     ·  144367     ·  144367     ·     10       ·   0.6       │
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G1Helper ·  _bn256CheckPairingBatch  ·  182830     ·  182866     ·  182854     ·      5       ·   0.8       │
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G1Helper ·  _isOnCurve               ·   24119     ·   25104     ·   24885     ·      9       ·   0.1       │
+·················|···························|·············|·············|·············|··············|··············
+|  BN256G1Helper ·  _isOnCurveSubsidized     ·   25053     ·   25821     ·   25667     ·     10       ·   0.1       │
```

Comment:
We included the gas analysis for G1 `isOnCurve` functions to show which one was cheaper. :P 

Maybe we should move the more expensive (`_isOnCurveSubsidized`) outside of the library as I suppose everyone would use the cheap one. However, I would keep the code to show the benchmark you did! :1st_place_medal:

Reply:
done! thanks for the suggestion